met-web/src/components/layout/Footer/index.tsx

```diff
@@ -226,6 +281,56 @@ const Footer = () => {
                         </Link>
                     </Stack>
                 </Grid>
+
+                {/* Footer admin login */}
+                <Grid item xs={12} sm={6} container justifyContent={'flex-start'} alignItems="baseline">
+                    <Unless condition={isLoggedIn}>
+                        <Grid item xs={12}>
+                            <Link
+                                href="/"
+                                onClick={(event) => {
+                                    event.preventDefault();
+                                    UserService.doLogin(baseURL);
+                                }}
+                                sx={linkStyles}
+                            >
+                                {translate('footer.login')}
+                                <LinkArrow />
+                            </Link>
+                        </Grid>
+                    </Unless>
+                </Grid>
+
+                {/* Footer version info */}
+                <When condition={isLoggedIn}>
+                    <Grid item container xs={12} sm={6} gap={1} justifyContent={{ xs: 'flex-start', sm: 'flex-end' }}>
+                        <VersionInfoDisplay
+                            label={translate('footer.appVersion')}
+                            version={AppConfig.version.version}
+                            branch={AppConfig.version.branch}
+                            commitUrl={AppConfig.version.commitUrl}
+                            buildDate={AppConfig.version.buildDate}
+                            isExpanded={isVersionExpanded}
+                            onToggle={() => setIsVersionExpanded(!isVersionExpanded)}
+                        />
+                        <When condition={isVersionExpanded}>
+                            <Suspense fallback={<BodyText>Loading...</BodyText>}>
+                                <Await resolve={loaderData?.apiVersion}>
+                                    {(apiVersionInfo: VersionInfo) => (
+                                        <VersionInfoDisplay
+                                            label={translate('footer.apiVersion')}
+                                            version={apiVersionInfo?.version}
```

Comment:
Do these properties need fallbacks? I'm guessing you already handled it, just checking.